src/containers/find-offer/offer-container/OfferContainer.tsx

```diff
@@ -0,0 +1,51 @@
+import useBreakpoints from '~/hooks/use-breakpoints'
+import { FC } from 'react'
+import Box from '@mui/material/Box'
+import Grid from '@mui/material/Grid'
+import { CardsViewTypes, OfferResponse, CardsViewEnums } from '~/types'
+import OfferCardSquare from '~/containers/find-offer/offer-card-square/OfferCardSquare'
+import OfferCard from '~/components/offer-card/OfferCard'
+
+interface OfferContainerProps {
+  viewMode: CardsViewTypes
+  offerCards: OfferResponse[]
+}
+
+const OfferContainer: FC<OfferContainerProps> = ({ viewMode, offerCards }) => {
+  const { isMobile, isDesktop } = useBreakpoints()
+  const onBookmarkClick = (id: string) => {
+    console.log(id)
+  }
+
+  const columnNumber = viewMode === CardsViewEnums.Grid ? 12 : 1
+
+  const renderSquareCard =
+    isMobile || (isDesktop && viewMode === CardsViewEnums.Grid) ? true : false
```

Comment:
you don`t need ternary here, isMobile || (isDesktop && viewMode === CardsViewEnums.Grid), this expression returns boolean

Reply:
Cleaned